src/test/scala/chiselTests/BundleSpec.scala

```diff
@@ -76,15 +77,24 @@ class BundleSpec extends ChiselFlatSpec with BundleSpecUtils {
   }
 
   "Bundles" should "not be able to use Seq for constructing hardware" in {
-    (the[ChiselException] thrownBy {
+    // This could either be an exception or an error message.
+    val thrown = the[ChiselException] thrownBy {
       elaborate {
         new Module {
           val io = IO(new Bundle {
             val b = new BadSeqBundle
           })
         }
       }
-    }).getMessage should include("Public Seq members cannot be used to define Bundle elements")
+    }
+    // Combine exception and (possible) error messages.
```

Comment:
Can this part be made less boilerplate-y? From the perspective of a test author this looks like an increase in complexity.

Reply:
Yes, I believe we can add some special `SubString extends String` types and the corresponding `areEqual` definition to ChiselTesters. I'll look into this.